api/envoy/api/v2/route/route.proto

```diff
@@ -727,6 +727,8 @@ message RouteAction {
   oneof host_rewrite_specifier {
     // Indicates that during forwarding, the host header will be swapped with
     // this value.
+    //
+    // [#next-major-version: host_rewrite_literal]
```

Comment:
What's the story with this tag? Is there an issue tracking this being implemented? I assume we want to change this for v3? cc @htuch

Reply:
I asked for @rgs1 to leave a bread crumb for us to do the rename.

Reply:
OK sounds good.